src/DataTable/DataTable.docs.json

```diff
@@ -0,0 +1,110 @@
+{
+  "id": "data-table",
+  "name": "DataTable",
+  "status": "draft",
+  "a11yReviewed": false,
+  "stories": [],
+  "props": [
+    {
+      "name": "aria-describedby",
+      "type": "string",
+      "description": "Provide an id to an element which uniquely describes this table"
+    },
+    {
+      "name": "aria-labelledby",
+      "type": "string",
+      "description": "Provide an id to an element which uniquely labels this table"
+    },
+    {
+      "name": "data",
+      "type": "Array<Data>",
+      "description": "Provide a collection of the rows which will be rendered inside of the table"
+    },
+    {
+      "name": "columns",
+      "type": "Array<Column<Data>>",
```

Comment:
It would be nice to explicitly render the `Column` type's shape somewhere

Reply:
Great point, let me see if I can type it out explicitly here 👀